From 2d655ee3bfa630d039be9748fa700e67c89e9bd2 Mon Sep 17 00:00:00 2001
From: Oliver Heger <oheger@apache.org>
Date: Thu, 11 Nov 2010 21:05:53 +0000
Subject: [PATCH] [CONFIGURATION-428] Updated changes.xml. Ported from trunk.
 However, the fix for this issue will not be ported because list handling is
 going to change anyway.

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/configuration/branches/configuration2_experimental@1034125 13f79535-47bb-0310-9956-ffa450edef68
---
 src/changes/changes.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/changes/changes.xml b/src/changes/changes.xml
index cda367edb4..becbefc00b 100644
--- a/src/changes/changes.xml
+++ b/src/changes/changes.xml
@@ -79,6 +79,10 @@
     </release>
 
     <release version="1.7" date="in SVN" description="">
+      <action dev="oheger" type="fix" issue="CONFIGURATION-428">
+        XMLConfiguration no longer escapes backslashs in the values of
+        XML elements.
+      </action>
       <action dev="oheger" type="fix" issue="CONFIGURATION-424">
         HierarchicalINIConfiguration now works correctly with configurations
         that contain only properties in the global section.
